CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 * Your contribution here.
 * [#115](https://github.com/radar/distance_of_time_in_words/pull/115): Use constants for time durations (2x faster, 3.5x less memory) - [@krzysiek1507](https://github.com/krzysiek1507)
+* [#117](https://github.com/radar/distance_of_time_in_words/pull/117): Support `#distance_of_time_in_words_to_now`. - [@joshuapinter](https://github.com/joshuapinter)
```

Comment:
It's support `vague: true`, right, not just support?

Move the period to the end from before ` - [@joshuapinter]` to match the format, same for the change line above.

Reply:
You're right! Done!